src/silicone/database_crunchers/latest_time_ratio.py

```diff
@@ -140,7 +140,6 @@ def get_values_in_key_timepoint(idf):
                     )
                     raise ValueError(error_msg)
                 else:
-                    # TODO: make interpolate method of IamDataFrame datetime friendly
```

Comment:
We haven't actually solved this. The todo was there to remind us that this behaviour should be natively supported in pyam rather than requiring us to pull out a dataframe with `.timeseries` before interpolating.

Reply:
I made the changes in pyam but didn't change the code here - good spot.